src/components/common/Spine/Loader.vue

```diff
@@ -9,7 +9,6 @@
 import { onMounted, watch } from 'vue'
 import { useMarket } from '@/stores/market'
 
-// @ts-ignore
```

Comment:
add this line back

Reply:
Whoops. While we're on it, any idea on why we are importing Spine this way? Were custom modifications made to it? Because there's newer versions and it's fully typed on NPM.

Reply:
Yes there were custom modifications made, from the top of my head: 

- Custom error message if someone tries to load a non existing asset ( eg "This character does not have this animation.." )
- a fix for transparency that had to be manually corrected, so I'm sure the npm packages have it broken
- a fix for the custom spine loader, where the base64 was wrongly decoded and resulted in, although insanely rare occasions, being impossible to load assets

Also nikke uses assets from Spine 4.0 and Spine 4.1 ( I have to update all the old 4.0 to 4.1 one day for better quality ) , thus if I import spine through npm I don't think I can have both runtimes/loaders. And of course spine 4.2 will never be able to run 4.0 and 4.1 assets